appendices/spirvenv.txt

```diff
@@ -304,6 +304,18 @@ ifdef::VK_EXT_shader_demote_to_helper_invocation[]
 endif::VK_EXT_shader_demote_to_helper_invocation[]
 |====
 
+[NOTE]
+.Note
+====
+1::
+    <<features-shaderStorageImageExtendedFormats,pname:shaderStorageImageExtendedFormats>>
+    feature is not strictly necessary to be enabled to use
+    code:StorageImageExtendedFormats SPIR-V capability.
+    _Extended storage image formats_ may: support
+    `VK_FORMAT_FEATURE_STORAGE_IMAGE_BIT` regardless of
+    pname:shaderStorageImageExtendedFormats being supported or enabled.
```

Comment:
I've been rolling this one around in my head and I don't think this note clarifies anything.  The feature doesn't have to be set because it's purely informative.  This is just like `textureCompressionETC2` which you don't have to enable to use ETC2 formats.  If anything, saying it's optional and adding a note is just going to confuse people.

Reply:
I think optionally enablable features are inherently confusing, given usual features do not work that way. If enabling it does nothing it should be noted _somewhere_. Maybe I could say that if it is supported, it is always enabled implicitly?

Reply:
I 100% agree  The text for `textureCompressionETC2` currently says:

> `textureCompressionETC2` specifies whether all of the ETC2 and EAC compressed texture formats are supported. If this feature is enabled, then the `VK_FORMAT_FEATURE_SAMPLED_IMAGE_BIT`, `VK_FORMAT_FEATURE_BLIT_SRC_BIT` and `VK_FORMAT_FEATURE_SAMPLED_IMAGE_FILTER_LINEAR_BIT` features must be supported in optimalTilingFeatures for the following formats:

However, those bits are queried by `vkGetPhysicalDeviceFormatProperties` which is a physical device function and therefore not subject to feature enables.  I think it's better to say that the features are implicitly enabled if they are available.  Maybe @jbreton-nv or @Tobski have an opinion here?

Reply:
ok I tried something else.